### PR TITLE
fixes #315 and adds a little clarity to the certificate related command instructions

### LINKED ISSUE #315: Unable to bootstrap with windows-winrm over ssl
I am trying to bootstrap a windows node with WinRM over ssl and it gives me a bare error. Any clues. I did the following steps before bootstrapping to configure ssl and that seem to work. Is this a known issue?

knife windows cert generate --cert-passphrase "test" --hostname "servername.domainname.local" --output-file "server_cert.pfx"
Generated Certificates:
- server_cert.pfx - PKCS12 format key pair. Contains public and private keys, can be used with an SSL server.
- server_cert.b64 - Base64 encoded PKCS12 key pair. Contains public and private keys, used by some cloud provider API's to configure SSL servers.
- server_cert.pem - Base64 encoded public certificate only. Required by the client to connect to the server.
Certificate Thumbprint: d05CdasdadadadadaCFC6

c:\Chef\devops>knife windows cert install "server_cert.pfx" --cert-passphrase "test"
Adding certificate to the Windows Certificate Store...
Certificate added to Certificate Store

c:\Chef\devops>knife windows listener create --cert-passphrase "test" --hostname "servername.domainname.local" --cert-thumbprint "d05CdasdadadadadaCFC6"
WinRM listener created with Port: 5986 and CertificateThumbprint: d05CdasdadadadadaCFC6

knife bootstrap windows winrm -t ssl  servername.domainname.local -x "admin" -P "adminpass"   -f  "server_cert.pfx"
It gives me the following error 
ERROR: OpenSSL::X509::StoreError:

### PR DESCRIPTION
**Pull request by mwrock**
This attempts to address some ambiguity in the `README` language around the new certificate commands. Specifically making it more clear where commands should be run (node or workstation) and which certificate format to use with the `-f` argument to bootstrapping or `knife winrm` commands.

### CODE REVIEW

**Comment by mwrock** [comment]
cc @chef/client-windows

File: README.md [hunk 1]
```diff
@@ -84,10 +84,11 @@ An initial run_list for the node can also be passed to the subcommand. Example u
...
-The PKCS12(i.e *.pfx) contains both the public and private keys, usually used on the server. This will be added to WinRM Server's Certificate Store.
-The *.b64 is Base64 PKCS12 key pair. Contains both the public and private keys, for upload to the Cloud REST API. e.g. Azure.
-The *.pem is Base64 encoded public certificate only. Required by the client to connect to the server.
+The certificate will be generated in three different formats:
+* **.pfx** - The PKCS12(i.e .pfx) contains both the public and private keys, usually used on the server. This can be added to a WinRM Server's Certificate Store using `knife windows cert install` (see command description below). **Note:** Do not use the *.pfx file with the `-f` argument on the `knife bootstrap windows winrm` and `knife winrm` commands. Use the *.pem file instead.
+* **.b64** - The *.b64 is Base64 PKCS12 key pair. Contains both the public and private keys, for upload to the Cloud REST API. e.g. Azure.
+* **.pem** - The *.pem is Base64 encoded public certificate only. One can use this file with the `-f` argument on `knife bootstrap windows winrm` and `knife winrm` commands.
```
Comment (btm) [inline README.md]: maybe this should be listed first?

**Comment by btm** [comment]
:+1: but consider putting pem first in the list.

**Comment by mwrock** [comment]
will do thanks @btm

**Pull request merged**